titus-common-ext/k8s/src/main/java/com/netflix/titus/ext/k8s/clustermembership/connector/K8ClusterMembershipConnectorComponent.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.ext.k8s.clustermembership.connector;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+import com.google.common.base.Preconditions;
+import com.netflix.titus.api.clustermembership.connector.ClusterMembershipConnector;
+import com.netflix.titus.api.clustermembership.model.ClusterMember;
+import com.netflix.titus.common.runtime.TitusRuntime;
+import com.netflix.titus.common.util.StringExt;
+import io.kubernetes.client.ApiClient;
+import io.kubernetes.client.util.ClientBuilder;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+
+@Configuration
+public class K8ClusterMembershipConnectorComponent {
```

Comment:
minor nit: just noticed these classes are missing the `s` in their prefix (`K8 => K8s`). It would be probably better to rename them to `Kube*` to be consistent with other classes in the `master.mesos.kubeapiserver` package.

Reply:
So rename `K8ClusterMembershipConnectorComponent` to `KubeClusterMembershipConnectorComponent`?